config.js

```diff
@@ -11,12 +11,142 @@ System.config({
 System.config({
```

Comment:
On workflow, the dependencies are locked in the [package.json](https://github.com/guardian/workflow/blob/master/prototype/package.json). On ci, `jspm install -y` will automatically generate the `config.js`.

Locking the dependencies ensured that we have consistent builds, as we had problems with dependencies updating without our knowledge and breaking our code - even patch releases.